### PR TITLE
Correct redirect for when nothing to submit

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/controllers/DeclarationController.scala [hunk 1]
```diff
@@ -77,10 +78,11 @@ class DeclarationController @Inject()(
 
       submitService.submitReport(request.pstr, data).flatMap { r =>
```
Comment (ngmjohnstone) [inline app/controllers/DeclarationController.scala:79]: What is `r` here? Could we rename?

Comment (grantrobson) [reply app/controllers/DeclarationController.scala]: done

**Review by ngmjohnstone — COMMENTED** [review]
Minor comment r.e. naming of function literal parameter

**Review by ngmjohnstone — APPROVED** [review]
Approved

**Pull request merged**